NEWS

```diff
@@ -1,3 +1,6 @@
+42.3.20170622
```

Comment:
Please don't edit NEWS like that -- I have a script that I run just before packaging that pulls the changes together automatically.